config/webpacker.yml

```diff
@@ -36,8 +36,8 @@ development:
   dev_server:
     https: false
     host: localhost
-    port: 5000
-    public: localhost:5000
+    port: 8080
+    public: localhost:8080
```

Comment:
I think we should only change from using the default ports if absolutely necessary. Is there a compelling reason for this change?